redshift_sqlalchemy/dialect.py

```diff
@@ -650,8 +653,7 @@ def _get_all_table_and_view_info(self, connection, **kw):
                 tables[key] = rel
             if rel.relkind == 'v':
                 views[key] = rel
-        self._all_tables_and_views = (tables, views)
```

Comment:
In an earlier draft of the reflection stuff, I was caching the result of the query here so that it would only run once, and the cached result would be used for further table reflections, but it was an overly aggressive approach since the dialect would never be able to adapt to changes in schema after the first reflection call. In particular, it broke the test suite where we were creating tables for the reflection tests.

These should absolutely be removed.